java-checks/src/test/files/checks/spring/SpringComponentWithNonAutowiredMembersCheck.java

```diff
@@ -0,0 +1,36 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.stereotype.Service;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class HelloWorld {
+
+  private String name = null; // Noncompliant [[sc=18;ec=22]] {{Make this member @Autowired or remove it.}}
+  public String address = null; // Noncompliant [[sc=17;ec=24]] {{Make this member @Autowired or remove it.}}
+  String phone = null; // Noncompliant [[sc=10;ec=15]] {{Make this member @Autowired or remove it.}}
```

Comment:
There is also the (nonsense I admit, but kinda important for coverage sake and thinking about how to make your code fail) case missing where a static var is annotated.